docs/workforce-management-framework/topics/pro-overview-introduction.adoc

```diff
@@ -0,0 +1,99 @@
+== Introduction
+
+{WFM-RC-NameLong}  consist of set of node.js modules written in TypeScript language that are packaged and distributed as
+{WFM-RC-NameLong} provides a set of reusable components that can simplify mobile, web and cloud application development.
+Solution allows developers to write secure, offline ready websites and mobile applications.
+{WFM-RC-NameLong} provides Workforce Management oriented framework for Workforce Management applications.
+
+Solution consist of two fully independent frameworks:
+
+ * Workflow Management Framework
+ * Rapid Application Development Framework
+
+As extension to frameworks above {ProductName} provides set of example module implementations as part of demo applications.
+
+===  Workflow Management Framework
+
+{WFM-RC-NameLong} Workflow Management Framework build BPMN compliant workflow management solution.
+WFM framework allows to build advanced Business Process Management solutions.
+Framework helps to write solutions that connects a business back-office with its fleet of mobilized employees.
+
+ * Task: defines single unit of work for business process
+ * Flow: contain ordered set of tasks and handlers for executing tasks in proper order
+ * Process: instance of flow containing current progress and business objects.
+
+Users can write custom Tasks (Workflow Steps) that will implement any business logic and link implementation with any UI solution.
+Framework abstract from any web or mobile application libraries, allowing developers to focus on implementation of the process.
+
+===  Rapid Application Development Framework
+
+{WFM-RC-NameLong} Rapid Application Development Framework allows to build production-grade Node.js and Cordova based applications.
+
+image:../../shared/images/technologies.png[Technologies]
+
+Main targets for framework:
+
+* Rapid, production ready application development framework.
+* Covers Mobile, web and server side application development
+* Written in TypeScript (with support JavaScript development)
+* Secure out of the box thanks to multiple security providers (Keycloak, Passport.js)
+* Full support for offline enabled applications thanks to FeedHenry Sync
+* Extensibility provided by set of TypeScript based interfaces
+* Provide A sleek, modern, mobile enabled front-end reference applications
+* Provides abstraction layer for common application elements:
+    ** Data access
+    ** Security
+    ** Logging
+    ** Restful API creation
+    ** Unit testing
+
+Framework abstract from any application logic and technologies allowing developers to extend it for their needs.
+
+===  Demo application
+
+- Set of example implementations for Workflow Management Solutions (Workorder, WorkorderResult)
```

Comment:
Let's mention that we have `Demo purpose examples etc.` not production ready. Just to showcase modules.